integration/cri-o/cri-o.sh

```diff
@@ -16,6 +16,11 @@ crio_repository="github.com/kubernetes-incubator/cri-o"
 crio_version=$(get_version "externals.crio.version")
 check_crio_repository="$GOPATH/src/${crio_repository}"
 
+if [ "$ID" == "centos" ]; then
+	echo "Skip - CRI-O tests are not supported yet"
```

Comment:
Can you create an issue and add it here? so we dontt forget about re-enabling them later.